.circleci/config.yml

```diff
@@ -1253,6 +1253,7 @@ jobs:
       - image: cimg/go:1.17
         environment:
           GO111MODULE: "on"
+    resource_class: large
```

Comment:
in a follow up, i will try to see why golangci-lint is suddenly having trouble completely in time (happening on master, too)